src/genie/libs/parser/asa/tests/ShowInventory/cli/equal/golden_output_expected.py

```diff
@@ -1,26 +1,26 @@
 expected_output = {
-    "Chassis": {
-        "description": "ASA 5555-X with SW, 8 GE Data, 1 GE Mgmt",
-        "pid": "ASA5555",
-        "vid": "V01",
-        "sn": "AAAAA11111",
+    'Chassis': {
```

Comment:
please don't switch the quoting of the whole unit test

Reply:
The reason I did because I normally keep single quote to differentiate between the json object or python dict, but it doesn't really matter here. Happy with whatever you go with :)